weave/flow/eval.py

```diff
@@ -137,7 +137,7 @@ def model_post_init(self, __context: Any) -> None:
         if self.name is None and self.dataset.name is not None:
             self.name = self.dataset.name + "-evaluation"  # type: ignore
 
-    @weave.op()
```

Comment:
Is there a functional reason this was this changed from `@weave.op()` to `weave.op`? Some of the others were left as calls so it's not immediately apparent to me

Reply:
just stylistic -- it's cleaner without paren imo

Reply:
Honestly agree